tests/setupTests.ts

```diff
@@ -9,7 +9,3 @@ global.IS_REACT_ACT_ENVIRONMENT = true
 
 // Mock scheduler to test React features
 vi.mock('scheduler', () => require('scheduler/unstable_mock'))
-
-// Silence react-dom & react-dom/client mismatch
-const logError = global.console.error.bind(global.console.error)
-global.console.error = (...args: any[]) => !args[0].startsWith('Warning') && logError(...args)
```

Comment:
This was R3F during the transition period in react 18 alpha because logs can halt CI. We should probably remove it there as well.

Reply:
https://github.com/pmndrs/react-three-fiber/pull/2491